docs/lib/auth/fragments/android/hub_events/10_listen_events.md

```diff
@@ -48,4 +48,37 @@ Amplify.Hub.subscribe(HubChannel.AUTH) { hubEvent: HubEvent<*> ->
 ```
 
  </amplify-block>
+ <amplify-block name="RxJava">
+
+```java
+RxAmplify.Hub.on(HubChannel.Auth)
+      .map(hubEvent::getName)
+      .flatMapObservable(
```

Comment:
```suggestion
      .map(
```

I pasted this code in a sample app. Apparently it's already an observable coming out of the `on` method.